test/csuite/timestamp_abort/main.c

```diff
@@ -272,6 +272,8 @@ thread_run(void *arg)
 	 * are in use.
 	 */
 	use_prep = (use_ts && td->info % 2 == 0) ? true : false;
+	use_prep = false;
```

Comment:
This was just to avoid thinking about `prepare_transaction` -- AFAIK, it shouldn't matter for the purposes of measuring the cost of accessing the `commit_timestampq`.